src/borg/archiver/rlist_cmd.py

```diff
@@ -23,7 +23,7 @@ def do_rlist(self, args, repository, manifest):
             format = "{archive}{NL}"
         else:
             format = os.environ.get("BORG_RLIST_FORMAT", "{archive:<36} {time} [{id}]{NL}")
-        formatter = ArchiveFormatter(format, repository, manifest, manifest.key, json=args.json, iec=args.iec)
+        formatter = ArchiveFormatter(format, repository, manifest, manifest.key, iec=args.iec)
```

Comment:
does this also need a change to the `.format_item()` call?

Reply:
I did the change in previous commits. And I'm sure that pytest passed